### DISCUSSION TITLE
Xdrip+ watchfaces not showing in Galaxy Watch 6 using Wear OS 4.0

### DISCUSSION THREAD

**Discussion by shock9191** [NightscoutFoundation/xDrip#3003] [Bug reports]
Xdrip+ works excessively well with Galaxy Watch 4. Unfortunately, when the app is sideloaded into a Galaxy Watch 6, the associated watchfaces aren't accessible in any way. 

Anyone has found how to access them ?

**Comment by agmsd** [comment]
I'm having the same problem with my Galaxy watch 4 after upgrading to wear os 4.0!

**Reply by Igor-Arefev** [reply]
Does anyone know how to install xdrip on galaxy watch 6,Wear os 4? I think this question will affect a lot of people after upgrading to version 4 of the system

**Reply by NoChance777** [reply]
I used wear installer 2, worked just fine for me. Strictly follow the instructions and you are good

**Comment by rsbiserra** [comment]
> Xdrip + funciona excessivamente bem com o Galaxy Watch 4. Infelizmente, quando o aplicativo é transferido para um Galaxy Watch 6, os watchfaces associados não ficam acessíveis de forma alguma.
> 
> Alguém descobriu como acessá-los?

Pode me enviar os arquivos que funcionavam no watch 4? Estou começa a usar relógio e estou perdida.  O arquivo que instalei não aparece as watchfaces

**Comment by kaz45242** [comment]
I just sidelined xdrip+ onto galaxy watch 6 through galaxy s22 using bugjaeger. I tried wear installer2, manual adb connection, but could not succeed. Bugjaeger worked fine. It took only 5 min.

**Reply by icnich** [reply]
Bought bugjaeger and I'm still unsuccessful. Never had issues with WI2 on my galaxy watch 4, maybe I'm being too bold trying with the 7. What did you do differently?

**Comment by kaz45242** [comment]
I don't know why you cannot do.  I'm still using watch 6 with xdrip.  I
just recommend reset watch once again, and restart bugjaeger over again.  I
hope you could bugjaeger' next step, which shows multiple program list to
chose for installation to the watch.  I tried several application, but only
bugjaeger worked.
